Examples/Atlas/Sources/AtlasExamples/main.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import MongoSwift
+import NIO
+
+/// Atlas connection string examples
+
+func scramExample() throws {
+    // START SCRAM EXAMPLE HERE
+    let elg = MultiThreadedEventLoopGroup(numberOfThreads: 4)
+    let client = try MongoClient(
+        "mongodb+srv://<user>:<password>@<host>/<dbname>?retryWrites=true&w=majority",
```

Comment:
The atlas UI currently wants drivers to use the default db here, but it's really more of a shell-ism since it's basically just a less clear `authSource` for drivers. I left it in for consistency with other drivers' examples, but I think this should probably be changed.

Reply:
hmmm, yeah weird. I'm not sure what the correct way to bring that up would be / who to ask about it - @mbroadst do you know?
anyway I agree we should go with consistency with other drivers for now.

Reply:
A good start might be to @mention Pia in the ticket

Reply:
I left a comment on the SWIFT ticket explaining the situation.

Reply:
Pia responded saying to keep the dbname in there for consistency with other driver examples, but that they'll look into updating it for all examples.